src/vorta/scheduler.py

```diff
@@ -58,6 +59,27 @@ def __init__(self):
         # connect signals
         self.app.backup_finished_event.connect(lambda res: self.set_timer_for_profile(res['params']['profile_id']))
 
+        # connect to `systemd-logind` to receive sleep/resume events
+        # The signal `PrepareForSleep` will be emitted before and after hibernation.
+        service = "org.freedesktop.login1"
+        path = "/org/freedesktop/login1"
+        interface = "org.freedesktop.login1.Manager"
+        name = "PrepareForSleep"
+        if QtDBus.QDBusConnection.systemBus().interface().isServiceRegistered(service).value():
+            self.bus = QtDBus.QDBusConnection.systemBus()
+            self.bus.connect(service, path, interface, name, "b", self.loginSuspendNotify)
+        else:
+            logger.warn('Failed to connect to DBUS interface to detect sleep/resume events')
+
+    @QtCore.pyqtSlot(bool)
+    def loginSuspendNotify(self, data: bool):
```

Comment:
The var name `data` isn't very descriptive either. Maybe somthing along the lines of `suspend` or `begin`.